lsp/markdown/completion/configGet.md

```diff
@@ -0,0 +1,9 @@
+Snippet for getting a config variable.
```

Comment:
Same as elsewhere - the snippet shouldn't describe that it's a snippet, it should describe what a config variable is/does.